packages/ai/tests/lazy-tools.test.ts

```
@@ -0,0 +1,60 @@
+import { describe, it, expect } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)